src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <Description>Stackdriver .NET Exporter for OpenTelemetry.</Description>
     <PackageTags>$(PackageTags);Stackdriver;Google;GCP;distributed-tracing</PackageTags>
     <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Cloud.Monitoring.V3" Version="1.1.0-beta02" />
-    <PackageReference Include="Google.Cloud.Trace.V2" Version="1.0.0-beta02" />
+    <PackageReference Include="Google.Cloud.Monitoring.V3" Version="2.1.0" />
+    <PackageReference Include="Google.Cloud.Trace.V2" Version="2.0.0" />
     <PackageReference Include="OpenTelemetry" Version="0.2.0-alpha.404" />
```

Comment:
can you update to the latest here?

Reply:
@cijothomas , should we update? If yes, we will lose support for net452.